networks/ethereum/wallet.csv

```diff
@@ -1,14 +1,28 @@
 slug,provider,openSource,supportedPlatforms,custodial,mfa,msig,hardware,keyExport,native,evm,tendermint,tokensSupport,staking,price,support,audit,languages,starred,actionButtons
-metamask,!provider:metamask,,,,,,,,,,,,,,,,,,
-bifrost,!provider:bifrost,,,,,,,,,,,,,,,,,,
-ledger,!provider:ledger,,,,,,,,,,,,,,,,,,
-solidifi,!provider:solidifi,,,,,,,,,,,,,,,,,,
-oxenflow,!provider:oxenflow,,,,,,,,,,,,,,,,,,
-safepal,!provider:safepal,,,,,,,,,,,,,,,,,,
-tokenpocket,!provider:tokenpocket,,,,,,,,,,,,,,,,,,
-coolwallet,!provider:coolwallet,,,,,,,,,,,,,,,,,,
-dcent,!provider:dcent,,,,,,,,,,,,,,,,,,
-rabby,!provider:rabby,,,,,,,,,,,,,,,,,,
-trustwallet,!provider:trustwallet,,,,,,,,,,,,,,,,,,
-zerion,!provider:zerion,,,,,,,,,,,,,,,,,,
-phantom,!provider:phantom,,,,,,,,,,,,,,,,,,
+metamask,!provider:metamask,TRUE,"[""Web"", ""Browser Extension"",""iOS"",""Android""]",FALSE,FALSE,FALSE,,,,,,,,,,,,,[Open Source](https://docs.metamask.io/services/reference/ethereum); [Supported Platform]( https://metamask.io/faqs); [Custodial]( https://support.metamask.io/start/metamask-is-a-self-custodial-wallet/)
```

Comment:
No need to update data on custodial, mfa and msig - it is already present
The buttons is not set correctly from syntax perspective - please take a look how it is done here (https://github.com/Chain-Love/chain-love/blob/main/networks/filecoin/wallet.csv)

Reply:
Here, and on every line till line 12

Reply:
Also on lines 17 and 18